### DISCUSSION TITLE
Lab 1 Part 2 Feedback

### DISCUSSION THREAD

**Discussion by shivanidoshi26** [snlee159/Interactive-Lab-Hub#15] [General]
Wow, the writeup and work you put into this was so thorough. I really like that it's a wearable now and it vibrates rather than making a sound, so it maintains the fact that it's a SILENT study buddy. Overall, I think everything looks really great. 
- It would be awesome if the device was smaller, so it's not as clunky - but I understand that you were limited by the type of device you could demo the functionality with. 
- Maybe you could change the UI a bit to allow the user to customize the amount of time that the alarm is set for, instead of just predefined ones. 
- It would've been nice to see the functionality of the "time left" feature in the demo as well.
Great work, Sam!

**Comment by singhaniasnigdha** [comment]
I am so impressed with the amount of thought and work put into the prototype - from the storyboard to the different versions of the 'Silent Buddy'! I personally would prefer the wearable as that way I would not be disturbing the people I am studying with.

The vibration was an apt addition to the color alarm, as that way it will more distinctly attract the user's attention. 

Random thought - Wondering if we could build on this to develop something similar which will be synchronized across a remote study group!

**Comment by anam884** [comment]
Great work Sam! Incorporating previous feedbacks and making it wearable and adding vibrations was smart!  

The feedback I have is maybe the size and type of the wearable can be experimented more. Maybe you can prototype it as a finger wearable rather than a wrist to make it minimal yet effective. For the interface maybe you can have a dial rather than a touch interface to make it more compact and reduce the size.
